components/css/content/_contentPage.scss

```diff
@@ -52,11 +52,41 @@
 					width:100%;
 					margin-bottom:20px;
 				}
```

Comment:
If we're going to use colored bullets and numbers we have to do some trickery. I havent tested fallbacks for this yet.